packages/core/src/hierarchy.ts

```diff
@@ -92,4 +91,18 @@ export class Hierarchy {
       }
     }
   }
+
+  private addAncestors<T extends Obj>(_class: Ref<Class<T>>): void {
+    let cl: Ref<Class<Obj>> | undefined = _class
+    while (cl !== undefined) {
+      const list = this.ancestors.get(_class)
+      if (list === undefined) {
+        this.ancestors.set(_class, [cl])
+      } else {
+        list.push(cl)
+      }
+      const attrs = this.classes.get(cl)
+      cl = attrs?.extends
```

Comment:
What if A extends B
And user add B and when A?

We should probably check all available descendants on moment of adding A?

Reply:
If user try add class A extends class B when B is not exist will be throw error `const attrs = this.classes.get(cl)` here, and I think this is normal

Reply:
And we can not check descendants because we use ancestors for create descendants